spec/models/customer_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Customer, type: :model do
+  describe 'relationships' do
+    it { should have_many :items }
```

Comment:
To be more precise, this test could be `it { should have_many(:items).through(:customer_items) }`